cli/src/commands/config/list.rs

```diff
@@ -129,6 +131,17 @@ fn config_template_language(
         let out_property = self_property.map(|annotated| annotated.value.decorated("", ""));
         Ok(L::wrap_config_value(out_property))
     });
+    language.add_keyword("origin", |self_property| {
+        let out_property = self_property.map(|annotated| {
+            let source = annotated.source;
+            if let Some(path) = &annotated.path {
+                format!("[{source}:{}]", path.to_string_lossy())
+            } else {
+                format!("[{source}]")
+            }
```

Comment:
it's probably better to add separate template keywords (such as `source_type` and `source_path`)

Reply:
I thought about it, but I considered the path to really be a "property" of the User and Repo ConfigSource enums---splitting these apart means you have an extra keyword to deal with and need to handle the empty case. I think combining them into a single `origin` keyword is a clean approach so I'd rather go with that unless you insist.

Reply:
Let's say user wants to query config value and its file path, the most intuitive way is to use `-T'value ++ "\t" ++ source_path ++ "\n"'` or something. If `source` type/path was formatted in certain form, it would have to be split by string function. That seems a bit annoying.